docs/detections/prebuilt-rules-update-modified-unmodified.asciidoc

```diff
@@ -0,0 +1,116 @@
+[[prebuilt-rules-update-modified-unmodified]]
+== Update modified and unmodified Elastic prebuilt rules
+
+.Requirements
+[sidebar]
+--
+You must have an https://www.elastic.co/subscriptions/cloud[Enterprise] subscription to access this feature. If you have a Platinum subscription or lower, follow the guidelines in <<update-prebuilt-rules>> instead.
+--
+
+This page provides instructions for updating modified and unmodified prebuilt rules. You can also find information about <<rule-field-update-statuses,statuses>> or <<resolve-reduce-rule-conflicts,conflicts>> that you might encounter when updating rules. 
+
+To update rules:
+
+. Find *Detection rules (SIEM)* in the navigation menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+. In the *Rules* table, select the *Rule Updates* tab.
++
+NOTE: The *Rule Updates* tab doesn't appear if all your installed prebuilt rules are up to date.
++
+[role="screenshot"]
+image::images/prebuilt-rules-update.png[The Rule Updates tab on the Rules page]
```

Comment:
This screenshot is a bit outdated now with the new filters on the update table, would be nice to get a new one, maybe including the "modified" and "conflict" columns

Reply:
Good idea - updated in [a503d53](https://github.com/elastic/security-docs/pull/6568/commits/a503d53e214029b000e7a5fbb79c1d37c1405c16). Thanks!